packages/wallet-connect-sdk-wallet-core/src/sdk.ts

```diff
@@ -43,6 +43,7 @@ export class WcWalletSDK {
   private _proposals: TSessionProposal[] = []
   private _requests: TSessionRequest[] = []
   private _status: EStatus = EStatus.NOT_STARTED
+  private _permissibleWalletRequestMethods: Record<string, (args: TAdapterMethodParam) => Promise<any>>
```

Comment:
I think we can call it `nonAdapterMethods`, it's not pretty, but it's very clear.